### DISCUSSION TITLE
Boosting Awareness for ThingsDB

### DISCUSSION THREAD

**Discussion by joente** [thingsdb/ThingsDB#369] [General]
We've noticed that the [GitHub project star count](https://github.com/thingsdb/ThingsDB) isn't as high as we'd like. To generate more interest, what strategies can we implement?

We recently finished an ebook that dives deeper into ThingsDB's functionalities. Maybe this could be a good way to introduce new users to the project?

I'm open to any ideas on how we can grow the ThingsDB community together.

**Comment by stefanak-michal** [comment]
I finished demo to show javascript driver used from browser and I wrote article about it. It explains some basics of ThingsDB and my approach to development of that demo. https://dev.to/stefanak-michal/let-me-explain-a-thingsdb-todo-app-demo-2n9g

**Reply by joente** [reply]
I'm sorry for missing your August article. I was likely offline on vacation and didn't catch up afterward.

**Comment by rickmoonex** [comment]
Thinking about doing a video series on getting started with ThingsDB. Basically 'translating' the ThingsDB book into video...

**Reply by joente** [reply]
That would be great! Video content tends to be more effective for most people.
